Dangerfile

```diff
@@ -192,6 +204,48 @@ module VSPDanger
     end
   end
 
+  class GemfileProtector
```

Comment:
What about something less scoped for the name? `GemfileEnterprisinator`? Also a comment before the class describing it (e.g. # Prevents Sidekiq Enterprise from being removed from Gemfile).